src/store/modules/plioItems.js

```diff
@@ -1,4 +1,4 @@
-import PlioService from "@/services/PlioAPIService.js";
+import PlioService from "@/utils/APIservices/PlioAPIService.js";
```

Comment:
`APIService` not `Services`.

Reply:
Or `APIClients` would be a better name since I saw `rootAPIClient` below. Be consistent, one way or another

Reply:
Assuming API has it's own separate folder, and not inside utils/ , is this structure good enough?
```
/src
	- APIClients/
		- rootAPIclient.js
		- experimentService.js
		- userService.js
		- plioService.js
```

Reply:
Here, too - it should be `rootAPIClient` if we are going with it - needs to be consistent - that is the biggest thing.